R/prep_sankey.R

```diff
@@ -49,47 +45,56 @@ prep_sankey <- function(data_alignment,
       .data[["year"]] == .env[["year"]]
     )
 
-  if (is.null(middle_node2)) {
-    data_out <- data_alignment %>%
-      dplyr::mutate(
-        is_aligned = dplyr::case_when(
-          alignment_metric >= 0 ~ "Aligned",
-          alignment_metric < 0 ~ "Not aligned",
-          TRUE ~ "Unknown"
-        ),
-        middle_node = !!rlang::sym(middle_node)
-      ) %>%
-      dplyr::select(group_var, "middle_node", "is_aligned", "loan_size_outstanding") %>%
-      dplyr::group_by(!!rlang::sym(group_var), .data[["middle_node"]], .data[["is_aligned"]]) %>%
-      dplyr::summarise(loan_size_outstanding = sum(.data[["loan_size_outstanding"]], na.rm = TRUE)) %>%
-      dplyr::ungroup() %>%
-      dplyr::arrange(!!rlang::sym(group_var), .data[["is_aligned"]])
-  } else {
-    data_out <- data_alignment %>%
-      dplyr::mutate(
-        is_aligned = dplyr::case_when(
-          .data[["alignment_metric"]] >= 0 ~ "Aligned",
-          .data[["alignment_metric"]] < 0 ~ "Not aligned",
-          TRUE ~ "Unknown"
-        ),
-        middle_node = !!rlang::sym(middle_node),
-        middle_node2 = !!rlang::sym(middle_node2)
-      ) %>%
-      dplyr::select(group_var, "middle_node", "middle_node2", "is_aligned", "loan_size_outstanding") %>%
-      dplyr::group_by(!!rlang::sym(group_var), .data[["middle_node"]], .data[["middle_node2"]], .data[["is_aligned"]]) %>%
-      dplyr::summarise(loan_size_outstanding = sum(.data[["loan_size_outstanding"]], na.rm = TRUE)) %>%
-      dplyr::ungroup() %>%
-      dplyr::arrange(!!rlang::sym(group_var), .data[["is_aligned"]])
-  }
+  data_out <- data_alignment %>%
+    dplyr::mutate(
+      is_aligned = dplyr::case_when(
+        alignment_metric >= 0 ~ "Aligned",
+        alignment_metric < 0 ~ "Not aligned",
+        TRUE ~ "Unknown"
+      ),
+      middle_node = tools::toTitleCase(!!rlang::sym(middle_node))
```

Comment:
not sure, but I would've guessed that using `tools::toTitleCase()` would require adding `tools` to the `DESCRIPTION` file to avoid a R CMD check warning? (I know `tools` is included in base R, but I thought R CMD check was picky enough about that 🤷🏻)

Reply:
apparently not though, no notes in the CI checks